src/main/java/edu/greenblitz/robotName/utils/hid/SmartJoystick.java

```diff
@@ -14,7 +14,7 @@
  */
 public class SmartJoystick {
 	
-	private static final double DEADZONE = 0.05;
+	private static final double DEADZONE = 0.07;
```

Comment:
Oh no…

Reply:
the deadzone was so small it was causing issues so its now bigger

Reply:
That means our controllers are getting worse